src/components/CreateButton.tsx

```diff
@@ -1,24 +1,25 @@
 'use client';
 
 import { useState } from 'react';
+import { twJoin } from 'tailwind-merge';
```

Comment:
why not use twMerge instead? better for a long run
https://www.wsquare.io/blogs/how-to-use-and-get-benefits-from-tailwind-merge

Reply:
sure, I will do the refactor later when needed